### ISSUE TITLE
PatientCounts

### ISSUE THREAD

**Issue by bbimber** [immunoengineer/gliph#1]
Hello - this is an interesting tool.  One minor thing: i think your example paragraph has a typo.  I assume Patient and Counts are supposed to be separate columns.  In your example they are concatenated.

**Comment by weizhiting** [comment]
Yes, you are right.  In their supplementary pdf, patient and counts are separated.

**Comment by jacobglanville** [comment]
I have gone ahead and made the changes to the README to reflect this.
